WikidataItem.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+import importer_utils as utils
+from wikidataStuff.WikidataStuff import WikidataStuff as WDS
+import pywikibot
+
+
+DATA_DIR = "data"
+
+
+class WikidataItem(object):
+    """Basic data object for upload to Wikidata."""
+
+    def make_q_item(self, qnumber):
+        """Create a regular Item."""
+        return self.wdstuff.QtoItemPage(qnumber)
+
+    def make_pywikibot_item(self, value, prop=None):
```

Comment:
Done -- removed `prop`, added WbTime to this method and used it instead of the manual creation of WbTime items in "start date" qualifier and source timestamp.